_sources/canasta/DockerSettings.php

```diff
@@ -363,9 +363,14 @@ function isEnvTrue( $name ): bool {
 		// Enable SemanticMediaWiki first, because some Semantic extension don't work in other case
 		if ( isset( $dockerLoadExtensions['SemanticMediaWiki'] ) ) {
 			wfLoadExtension( 'SemanticMediaWiki' );
+			enableSemantics( $wgServer, true );
+		}
+		// Ensure PageForms is always enabled after SemanticMediaWiki
+		if ( isset( $dockerLoadExtensions['PageForms'] ) ) {
```

Comment:
Do we need to load PageFoms here separately? Won't it be loaded below "after SemanticMediaWiki" with other extensions? (if `|| $extension === 'PageForms'` removed?)

Reply:
Good catch!